neqo-transport/src/events.rs

```diff
@@ -168,7 +168,7 @@ impl ConnectionEvents {
 
     // The number of datagrams in the events queue is limited to max_queued_datagrams.
     // This function ensure this and deletes the oldest datagrams if needed.
-    fn check_datagram_queued(&self, max_queued_datagrams: usize) {
+    fn check_datagram_queued(&self, max_queued_datagrams: usize, stats: &mut Stats) {
         let mut q = self.events.borrow_mut();
         let mut remove = None;
         if q.iter()
```

Comment:
Before we ship this, we really need to ensure that we don't keep this bit.  We should just be able to do a simple comparison of `self.datagrams.len()` (or something similarly named) with a target value.

Reply:
Yes, open #1218 to fix this. That is the next on my list to fix.